nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -47,13 +47,36 @@ def __init__(self):
         self.client = client.CoreV1Api()
         self.client_apps = client.AppsV1Api()
         self.client_batch_api = client.BatchV1Api()
-        self.namespace = self.get_nuvlaedge_project_name(util.default_project_name)
-        self.job_engine_lite_image = os.getenv('NUVLAEDGE_JOB_ENGINE_LITE_IMAGE') or self.current_image
+        self.namespace = \
+            self.get_nuvlaedge_project_name(util.default_project_name)
+        self.job_engine_lite_image = \
+            os.getenv('NUVLAEDGE_JOB_ENGINE_LITE_IMAGE') or self.current_image
         self.host_node_ip = os.getenv('MY_HOST_NODE_IP')
         self.host_node_name = os.getenv('MY_HOST_NODE_NAME')
-        self.vpn_client_component = os.getenv('NUVLAEDGE_VPN_COMPONENT_NAME', 'vpn-client')
+        self.vpn_client_component = \
+            os.getenv('NUVLAEDGE_VPN_COMPONENT_NAME', 'vpn-client')
+        self.set_image_pull_policy = self.checked_image_pull_policy()
```

Comment:
This is specific for `job` so this variable should represent that.
```python
        self.job_image_pull_policy = self.checked_image_pull_policy()
```
.
The function/method should also represent that 
```suggestion
        self.job_image_pull_policy = self.get_job_image_pull_policy()
```
or maybe better, the `getenv` should be extracted to make the function/method generic
```suggestion
        job_img_pull_policy = os.getenv('JOB_IMAGE_PULL_POLICY')
        self.job_image_pull_policy = self.get_image_pull_policy(job_img_pull_policy)
```

Reply:
And as you can see in my second "suggested change", I would rename the `JOB_ENGINE_IMAGE_PULL_POLICY` to `JOB_IMAGE_PULL_POLICY` to be shorter and match the format we have on _docker-compose.yml_ file for docker NE.

Reply:
I've changed the name to reflect the job image policy.
Changed the call to the checked_image_pull_policy
Added the default constant